src/fides/api/ops/service/messaging/message_dispatch_service.py

```diff
@@ -35,6 +40,47 @@
 logger = logging.getLogger(__name__)
 
 
+def check_and_dispatch_error_notifications(db: Session) -> None:
+    privacy_request_notifications = PrivacyRequestNotifications.all(db=db)
+    if not privacy_request_notifications:
+        return None
+
+    unsent_errors = PrivacyRequestError.filter(
+        db=db, conditions=(PrivacyRequestError.message_sent.is_(False))
+    ).all()
+    if not unsent_errors:
+        return None
+
+    email_config = CONFIG.notifications.notification_service_type in (
+        MessagingServiceType.MAILGUN.value,
```

Comment:
we have an existing const `EMAIL_MESSAGING_SERVICES` that you can use here- includes both mailgun and twilio email

Reply:
Nice, thanks for pointing this one out